setup-env/action.yaml

```diff
@@ -11,6 +11,9 @@ runs:
         if [ ! -e .trunk/trunk.yaml ]; then
           ${TRUNK_PATH:-trunk} init
         fi
+        cat >>$GITHUB_ENV <<EOF
+        INPUT_ARGUMENTS=$INPUT_ARGUMENTS --output-file=.trunk/landing-state.json
+        EOF
```

Comment:
We should only do this in repo_tests; we don't want to automatically impose this on all customers